utils/data-processing-utils-worker-document/src/main/java/com/github/cafdataprocessing/utilities/worker/document/EmptyWorkerTask.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2015-2017 Hewlett Packard Enterprise Development LP.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafdataprocessing.utilities.worker.document;
+
+import com.hpe.caf.api.worker.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A minimal implementation of WorkerTask.
+ * These methods are here only to meet the interface contract on objects requiring WorkerTaskData
+ * and are not intended for use.
+ */
+public class EmptyWorkerTask implements WorkerTaskData {
+    private static final Logger LOGGER = LoggerFactory.getLogger(EmptyWorkerTask.class);
```

Comment:
I think this was already commented by Anthony, if this object is just to fulfil the requirement to provide an implementation and any of it's method are not intended to be used, UnsupportedOperationException should be thrown.
Rationale for that is that if something is using it, it will not get correct data and will fail somewhere else without clear indication why.

Reply:
As mentioned in EmptyConfigSource, should I move this into family task submitter?